Xamarin.Essentials/DeviceDisplay/DeviceDisplay.tizen.cs

```diff
@@ -0,0 +1,79 @@
+﻿using System;
+using System.Runtime.InteropServices;
+
+namespace Xamarin.Essentials
+{
+    public static partial class DeviceDisplay
+    {
+        [DllImport("libcapi-system-device.so.0", EntryPoint = "device_power_request_lock")]
+        static extern void RequestKeepScreenOn(int type = 1, int timeout = 0);
+
+        [DllImport("libcapi-system-device.so.0", EntryPoint = "device_power_release_lock")]
+        static extern void ReleaseKeepScreenOn(int type = 1);
+
+        static bool keepScreenOn = false;
+
+        static bool PlatformKeepScreenOn
+        {
+            get => keepScreenOn;
+            set
+            {
+                if (value)
+                    RequestKeepScreenOn();
+                else
+                    ReleaseKeepScreenOn();
+                keepScreenOn = value;
+            }
+        }
+
+        static DisplayInfo GetMainDisplayInfo()
+        {
+            var display = Platform.MainWindow;
+            return new DisplayInfo(
+                width: display.ScreenSize.Width,
+                height: display.ScreenSize.Height,
+                density: display.ScreenDpi.X / (DeviceInfo.Idiom == DeviceIdiom.TV ? 72.0 : 160.0),
+                orientation: GetOrientation(),
+                rotation: GetRotation());
```

Comment:
I see you re no longer using `Platform.GetFeatureInfo<int>("xxx")`. Was there a reason for originally using it?

Reply:
The value through `Platform.GetFeatureInfo<int>("xxx")` is a fixed value. 
I think the current way of using changing values is more appropriate.